### PR TITLE
Fixes marble recipes not existing, and makes mapspawned marble tables exist again

### PR DESCRIPTION
**Pull request by BlueWildrose**
## About The Pull Request

Title.

## Why It's Good For The Game

Yes.

## Changelog

:cl:
fix: You can now make marble tile and marble statues using marble again.
fix: Maps that were meant to have marble tables will no longer make them appear as sandstone tables.
/:cl:

### CODE REVIEW

**Comment by silicons** [comment]
wait for #5950 before merging @ maintainers

**Review by TheLordME — APPROVED** [review]

**Comment by BlueWildrose** [comment]
seems like there's nothing to fix here because other PRs have already fixed it so I'm just closing this

**Pull request closed without merging**
